content-security-policy/generic/eval-typecheck-callout-order.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <script src='/resources/testharness.js' nonce='abc'></script>
+    <script src='/resources/testharnessreport.js' nonce='abc'></script>
+</head>
+<body>
+    <title>Test for order of Type(evalInput) and host callout</title>
```

Comment:
Did you mean for this to go in the `<head>`?

Reply:
Yes.  Fixed.